src/main/kotlin/no/nav/syfo/narmestelederapi/NarmesteLederClient.kt

```diff
@@ -68,6 +68,30 @@ class NarmesteLederClient(
             )
         }
     }
+
+    suspend fun hentAlleNarmesteLedereForSykmeldtFraSyfoserviceStrangler(sykmeldtAktorId: String): List<NarmesteLederRelasjon> {
+        val accessToken = accessTokenClient.hentAccessToken(resourceId)
+        return client.get<List<NarmesteLeder>>("$endpointUrl/sykmeldt/$sykmeldtAktorId/allenarmesteledere") {
```

Comment:
Dette endepunktet burde forresten ikke hete "allenarmesteledere".
Egentlig burde det hete "narmesteleder" og så burde det som heter "narmesteleder" i dag hete evt. "narmesteleder/aktiv".
Men siden det navnet et tatt, hva med å endret det til "narmesteledere", aka uten "alle" i syfoservicestrangler?

Reply:
Grunnen til at den heter `alle` nå, er fordi jeg synes det er vanskelig å se forskjell på `leder` og `ledere`, men nå går det fortere å se forskjellen på de to endepunktene.